apps/vessel-history/features/map/map.hooks.ts

```diff
@@ -53,3 +64,93 @@ export interface Viewport extends LatLon {
   longitude: number
   zoom: number
 }
+
+export const openFishingMap = (datasets, allGFWIds, viewport) => {
+  // colors used in the map
+  const colors = ['#f4511f', '#33b679', '#f09300', '#ffea00', '#9ca4ff']
```

Comment:
I'd grab this from the ui-components list we are actually using in fishing-map
`LineColorBarOptions`